tools/Makefile

```diff
@@ -88,21 +88,29 @@ packaging.done: activate_python.sh
 	. ./activate_python.sh && python3 -m pip install packaging
 	touch packaging.done
 
-pytorch.done: activate_python.sh packaging.done
+numba.done: activate_python.sh
+ifeq ($(strip $(USE_CONDA)),)
+	. ./activate_python.sh && python3 -m pip install numba
+else
+	. ./activate_python.sh && conda install -y numba
```

Comment:
It seems that `conda` cannot automatically install a very new version of `numpy`. However, `fairscale` requires `numpy >= 1.22.0`, which will cause an incompatible issue eventually.

```
numba 0.55.1 requires numpy<1.22,>=1.18, but you have numpy 1.23.3 which is incompatible.
Successfully installed fairscale-0.4.9 numpy-1.23.3
```

Instead, `python3 -m pip install numba` seems to work well. It will install `numpy 1.23`.

Reply:
I got it. Thank you.